editoast/src/views/projection.rs

```diff
@@ -214,6 +214,22 @@ fn compute_space_time_curves(
     space_time_curves
 }
 
+/// Interpolate a time value between two positions
+pub fn interpolate(
+    start_pos: u64,
+    end_pos: u64,
+    start_time: u64,
+    end_time: u64,
+    pos_to_interpolate: u64,
+) -> u64 {
+    if start_pos == end_pos {
+        start_time
```

Comment:
What if `end_time` is different of `start_time`? Should we always return `start_time`? Or `end_time`? Or it's a sign of a bug somewhere else, then in this case, maybe we can `assert_eq!(start_time, end_time)` so it will panic (if we don't want to panic, we may at least want to log)?

EDIT: And then I see below that this function was only moved. Feel free to ignore (but moving code in a PR is sometimes an opportunity to improve old code 🤷)

Reply:
Thank you very much. As long as this PR blocks work of @jacomyal , I'm currently considering a fix on another PR if you don't mind ?